### PR TITLE
adds dev app and devtools command

### PR DESCRIPTION
**Pull request by m4ra**
**Describe your changes**
Added the management command `devtools` for creating fake data for testing API performance.
For testing locally, we can run the command from the root directory.
Usage examples:
```
       $ ./manage.py devtools
       $ ./manage.py devtools --projects 0
       $ ./manage.py devtools --plans 550
       $ ./manage.py devtools --ext-projects 100
```

**Tasks**
- [ ] PR name contains story or task reference
- [x] Steps to recreate and test the changes
- [x] Documentation (docs and inline)
- [ ] Tests (including n+1 and django_assert_num_queries where applicable)
- [x] Changelog

### CODE REVIEW

File: meinberlin/config/settings/dev.py [hunk 1]
```diff
@@ -23,7 +23,10 @@
 except ImportError:
     pass
 else:
-    INSTALLED_APPS += ("debug_toolbar",)
+    INSTALLED_APPS += (
+        "debug_toolbar",
+        "meinberlin.apps.dev",
```
Comment (goapunk) [inline meinberlin/config/settings/dev.py:28]: should we move this in it's own block? so it's not tied to the import of the debug_toolbar?

**Review by goapunk — APPROVED** [review]
Looks good to me, just a small question about the installed apps. Feel free to merge

File: meinberlin/config/settings/dev.py (hunk 1 above)
Comment (m4ra) [reply meinberlin/config/settings/dev.py:28]: i feel, it'd look more weird to have the installed_apps there twice. Also they are both dev related tools.

**Comment by m4ra** [comment]
@hom3mad3 with this new command you can create fake plans and projects in meinberlin. Check description and the `docs/api_caching` about how to use it.

**Pull request merged**